installer/bundle/bundle_skel.sh

```diff
@@ -1328,6 +1331,8 @@ case "$installMode" in
         ;;
 
     I)
+        install_tst="yes" # set variable to install tst upon exit
```

Comment:
nit by convention var name is caps

Reply:
Sorry! Forgot about that, fixed here:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/1fcd4dd0d1cef4e18b71d295a048a3e4b7412dc9/installer/bundle/bundle_skel.sh#L1336